isce2_topsapp/__main__.py

```diff
@@ -129,5 +133,84 @@ def main():
             aws.upload_file_to_s3(file, args.bucket, args.bucket_prefix)
 
 
+def gunw_burst():
+    parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+    parser.add_argument('--bucket')
+    parser.add_argument('--bucket-prefix', default='')
+    parser.add_argument('--dry-run', action='store_true')
+    parser.add_argument('--reference-scene', type=str, required=True)
+    parser.add_argument('--secondary-scene', type=str, required=True)
+    parser.add_argument('--image-number', type=int, required=True)
+    parser.add_argument('--burst-number', type=int, required=True)
+    parser.add_argument('--azimuth-looks', type=int, default=2)
+    parser.add_argument('--range-looks', type=int, default=10)
+    args = parser.parse_args()
+
+    ensure_earthdata_credentials(args.username, args.password)
+
+    ref_resp, sec_resp = get_asf_slc_objects([args.reference_scene, args.secondary_scene])
+
+    ref_params = BurstParams(
+        safe_url=ref_resp.properties['url'],
+        image_number=args.image_number,
+        burst_number=args.burst_number,
+    )
+    sec_params = BurstParams(
+        safe_url=sec_resp.properties['url'],
+        image_number=args.image_number,
+        burst_number=args.burst_number,
+    )
+
+    ref_burst, sec_burst = download_bursts([ref_params, sec_params])
+
+    intersection = ref_burst.footprint.intersection(sec_burst.footprint).bounds
+    asc = ref_burst.orbit_direction == 'ascending'
```

Comment:
```suggestion
    is_ascending = ref_burst.orbit_direction == 'ascending'
```
would make the intent of this variable much clearer.

Reply:
I've changed this, as well as the argument name in the function it is used in.